docs/knowledge-base/DevOps/pipelines/release-management.mdx

```diff
@@ -0,0 +1,112 @@
+   - A new branch is created for every major version that is smaller than the latest major version.
+   - Example:
+     - If a service's latest release version is `3.3.0` and a new bug fix needs to be merged to latest release with major version '2', 
+       a pull request (PR) will be created for both:
+       - `master` (represents 3.x.x) - only if the bug fix relevant to this release too
+       - `2.x.x` (to track the new version with the fix).
+     - Once merged, both branches will represent a release with the bug fix, while the `1.x.x` branch remains unchanged.
+
+## Service Release Flow
+
+![Releasing Flow](./img/release-management/release-on-repo.png)
+
+Explanation:
+
+1. The developer creates a PR
+
+2. On merging to master: 
+    - release-please takes action and opens a PR release with these changes:
+        - Updating package.json to the new release version
+        - Updating Changelog with the new version
+        - Updating helm (chart version)
+
+    :::info
+    The release version will be based on ([SemVer](https://semver.org/)) (according to the PR title)
+    :::
+
+3. On merging the release PR to master:
+    - release-please will create a new release
+
+4. On new release, an automate workflow will be triggered to:
+    - if it is a library repo -> push to NPM
+    - if it has Dockerfile -> build a docker image and push to registry
+    - if it has helm -> package the helm chart and push to registry
+
+**End State:**
+    - New release version based on SemVer
+    - master points to the new release
+    - NPM, Docker and Helm registries updated with the new release
+
+## Helm Repositories Strategy
+
+![Helming Strategy](./img/release-management/helm-repositories-strategy.png)
+
+The project includes three additional repositories for Helm charts and values:
+
+1. Service Repositories:
+    - Each service repository contains its own actual Helm chart.
+
+2. **`helm-charts`**:
+    - This repository contains packaged versions of all the Helm charts used across services.
+    - The Helm charts themselves are imported from a registry via OCI, meaning the repository
+      stores the packaged charts rather than the actual charts.
+    - This repository represents the Libot Version during the annual production deployment.
+
+3. **`helm-values`**:
+   - Stores Helm values for different environments.
+   - Contains branches for each environment (e.g., `stage`, `prod`).
+
+When changes are pushed to `master` in the service repos,  
+automatic PRs are created in both `helm-charts` and `helm-values`:
+    - Initially, an automatic PR is created in `helm-chart`.
+    - Once the PR is merged to master, an automatic stage PR is opened in `helm-values`
+    - Once the stage PR is merged, another automatic PR is created for the prod branch, 
+      facilitating the promotion of changes to production.
+    - Once the prod PR is merged, a new release (represents Libot version) will be created in:
+        1. stage branch
```

Comment:
Why? This could be done when merging to `stage`, each could have thier own version when `prod` is the only one affecting the whole calculated version based on `helm-charts` (which describes the wanted structure).
And this means the "sealed" version used should be the version of `helm-charts` referenced in the `prod` `helm-values`.

Reply:
But you want the stage to represent libot version too so the stage can't be versioned before the helm-charts released new version